src/drivers/mediatek/mt8195/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This commit message is completely wrong - you don't add any topology in this patch.
Please update it correspondingly.

Reply:
> Can you please add a small description of the target: what DSP it has (from code review I realized is Hifi4), how many cores, how many DAIs, DMAs, what rates are supported, formats, etc. ?
> There is a lot of code and I believe it would help reviewing.
> Thanks!

ok, I add those information in request comment.